### ISSUE TITLE
Unity extension is deprecated

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode-docs#5360]
The unity extension is now deprecated. More details here https://github.com/Unity-Technologies/vscode-unity-debug/issues/206#issuecomment-1131461978

We should have a note, or a banner in our Unity docs to make this clear https://code.visualstudio.com/docs/other/unity

@gregvanl what do you think?

**Comment by gregvanl** [comment]
@isidorn We don't recommend that vscode-unity-debug extension in the main section of the Unity topic, only as a Common Questions answer for debugging https://code.visualstudio.com/docs/other/unity#_how-can-i-debug-unity
We could remove that section or change it to say the extension is deprecated.
I think I'm in favor of just removing that Common Question item and also remove the extension query earlier in https://code.visualstudio.com/docs/other/unity#_unity-extensions, where the Debugger for Unity is returned in the Marketplace  query.
I don't think we want to add banners for deprecated extensions, unless the notification is important for the topic flow

**Comment by isidorn** [comment]
@gregvanl I agree, let's just remove if from the common questions section. And remove the extension query.
That should be enough. No banners are needed. Thanks and sorry for the slowish response...
